Jay-RecipesX/app/src/main/java/com/example/jay_recipesx/features/RecipeHome/Data/Remote/RecipeApi.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.jay_recipesx.features.RecipeHome.Data.Remote
+
+import com.example.jay_recipesx.features.RecipeHome.Data.Models.RecipeModel
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Headers
+
+interface RecipeApi {
+    @Headers(
+        "Content-Type: application/json",
+        "x-api-key: 6b129304251e43469b1faa962b367fed"
```

Comment:
is it good idea to pass api key publically?

Reply:
No it's not good idea to pass the API will update this